extensions/amp-iframe/0.1/amp-iframe.js

```diff
@@ -274,6 +276,26 @@ export class AmpIframe extends AMP.BaseElement {
     }
     this.attemptChangeHeight(newHeight);
   }
+
+  /**
+   * Builds a key-value object representing data-params-* attributes on the
+   * element.
+   * @returns {Object}
+   * @private
+   */
+  getDataParams_() {
+    const attributes = this.element.attributes;
+    const params = {};
+    for (let i = 0; i < attributes.length; i++) {
+      const attr = attributes[i];
+      const matches = attr.nodeName.match(/^data-param-(.+)/);
+      if (matches) {
+        const param = matches[1];
+        params[param] = attr.nodeValue;
```

Comment:
Makes sense. I think we should stay consistent of how we treat data attributes so calling `dashToCamelCase` on them sgtm.